backend/src/yeomanui.ts

```diff
@@ -73,7 +76,28 @@ export class YeomanUI {
     // on the other hand, we never look for newly installed generators...
 
     const promise: Promise<IPrompt> = new Promise(resolve => {
+      const cwd = path.join(os.homedir(), "projects");
```

Comment:
This should be configurable by the user (as a vscode settings)

Reply:
Issue created - https://github.com/SAP/yeoman-ui/issues/57